pythoncz_tests/models/github_test.py

```diff
@@ -0,0 +1,432 @@
+import re
+import warnings
+
+import pytest
+import requests
+import responses
+from werkzeug.contrib.cache import NullCache
+
+from pythoncz.models import github as github_module
+from pythoncz_tests.models import github_test_fixtures as fixtures
```

Comment:
Those are pretty much functions. Sometimes called multiple times in a test. Not sure if pytest fixtures would be a good fit.